lib/airbrake/rake/tasks.rb

```diff
@@ -67,8 +67,8 @@
 
       # Avoid loading the environment to speed up the deploy task and try guess
       # the initializer file location.
-      if initializer.exist?
-        load(initializer) unless Airbrake[:default]
+      if initializer.exist? && Airbrake[:default].is_a?(Airbrake::NilNotifier)
```

Comment:
1. Check for NilNotifier is hacky
2. I have no idea why you load initialized only when default notifier is NilNotifier